inference-engine/tests/functional/plugin/gna/pass_tests/fq_fusion_with_sigmoid.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ie_core.hpp>
+#include "ngraph_functions/builders.hpp"
+#include "common_test_utils/test_constants.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+
+namespace LayerTestsDefinitions {
+
+typedef std::tuple<
+    std::string,                        // Target device name
+    InferenceEngine::Precision,         // Network precision
+    size_t,                             // Input size
+    std::map<std::string, std::string>  // Configuration
+> fqFusionWithSigmoidParams;
+
+class FqFusionWithSigmoidTest : public LayerTestsUtils::LayerTestsCommon,
+    public testing::WithParamInterface<fqFusionWithSigmoidParams> {
+protected:
+    void SetUp() override {
+        InferenceEngine::Precision netPrecision;
+        std::map<std::string, std::string> config;
+        size_t inputSize;
+        std::tie(targetDevice, netPrecision, inputSize, config) = this->GetParam();
+        configuration.insert(config.begin(), config.end());
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+        auto input = ngraph::builder::makeParams(ngPrc, {{1, inputSize}});
+        float left = 0;
```

Comment:
I'd make fq parameters to be the test parameters, and select them to be more similar with real ones
for example after Sigmoid min, max values should be -1, 1
number of levels should be 65535 for int16 quantization

Reply:
FQ parameters were moved to test parameters.